### ISSUE TITLE
RDS MigrationResource CREATE_FAILED

### ISSUE THREAD

**Issue by mikejpeters** [serverless-stack/sst#2983]
Deploying a new stack with RDS fails when creating the `MigrationResource`:

```
no identity-based policy allows the cloudformation:DescribeStacks action
```

Logs:

```
|  Database db/MigrationHandler/ServiceRole/DefaultPolicy AWS::IAM::Policy CREATE_COMPLETE 
|  Database db/MigrationHandler AWS::Lambda::Function CREATE_COMPLETE 
|  Database db/MigrationResource Custom::SSTScript CREATE_FAILED Received response status [FAILED] from custom resource. Message returned: User: arn:aws:sts::******:assumed-role/mike-rpm-Database-dbMigrationHandlerServiceRole-******/mike-rpm-Database-dbMigrationHandler****** is not authorized to perform: cloudformation:DescribeStacks on resource: arn:aws:cloudformation:ca-central-1:******:stack/mike-rpm-Database/****** because no identity-based policy allows the cloudformation:DescribeStacks action
```

My stack:

```ts
import { RDS, StackContext } from "sst/constructs";

export function Database({  stack }: StackContext) {
  return new RDS(stack, "db", {
    engine: "postgresql13.9",
    defaultDatabaseName: "main",
    migrations: "packages/core/migrations",
    types: "packages/core/src/sql.generated.ts",
  });
}
```

I believe this issue was introduced in sst `2.13.3`. The deploy works fine on `2.13.2`. 

I also tested a few postgres/mysql engines and the results were the same.

**Comment by jayair** [comment]
That's weird. Can you open a thread in #help? We'll take a look.

**Comment by mikejpeters** [comment]
https://discord.com/channels/983865673656705025/1119328118952435712

**Comment by edgar971** [comment]
I had the same issue. Rolled back to 2.12.0 and everything worked again.

**Comment by edgar971** [comment]
I also had the same issue and was able to get it working with version 2.13.2.

**Comment by juliankrispel** [comment]
I am using `2.13.6` and I have this issue, looks like the bug is ongoing

**Comment by vasco-santos** [comment]
Also experiencing this on a deployment and needed to downgrade to `2.13.2`

**Comment by fwang** [comment]
Released in v2.16.0

Thanks @iloewensen.
